tests/print_status/conf/dm-task.yaml

```diff
@@ -30,7 +30,8 @@ mydumpers:
     threads: 4
     chunk-filesize: 0
     skip-tz-utc: true
-    extra-args: "--statement-size=5000"
+    statement-size: 4000
```

Comment:
Why change `5000` to `4000` here?

Reply:
Because the difference of the meaning of statement-size between mydumper and dumpling. dumpling only counts the data's size, not counting the joiners. So I shorten this to split it to more files.